### PR TITLE
[ARM] enable snapshot performance tests

### PR DESCRIPTION
**Pull request by dianpopa**
## Reason for This PR

Enable snapshot performance integration tests for aarch64.

**Obs**: this is created against this [PR](https://github.com/firecracker-microvm/firecracker/pull/2351) so you may be seeing more commits than expected. You can only take into account the last 4 commits.

## Description of Changes

`[Author TODO: add description of changes.]`

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

File: tests/integration_tests/performance/test_snapshot_perf.py [hunk 1]
```diff
@@ -68,6 +80,7 @@ def _test_snapshot_create_latency(context):
                 .format(DEFAULT_TEST_IMAGES_S3_BUCKET))
     artifacts = ArtifactCollection(_test_images_s3_bucket())
     firecracker_versions = artifacts.firecracker_versions()
+    assert len(firecracker_versions) > 0
```
Comment (luminitavoicu) [inline tests/integration_tests/performance/test_snapshot_perf.py:83]: Good catch!

File: tests/integration_tests/performance/test_snapshot_perf.py [hunk 2]
```diff
@@ -41,8 +42,14 @@
...
 # TODO: Update the table after fix. Target is < 5ms.
 LOAD_LATENCY_BASELINES = {
-    '2vcpu_256mb.json': 8,
-    '2vcpu_512mb.json': 8,
+    'x86_64': {
+        '2vcpu_256mb.json': 8,
+        '2vcpu_512mb.json': 8,
+    },
+    'aarch64': {
+        '2vcpu_256mb.json': 2,
```
[the remark this answers is not in the document]
Comment (sandreim) [reply tests/integration_tests/performance/test_snapshot_perf.py:62]: Test failed : ```assert 2 > 2.03 
E   AssertionError: LoadSnapshot latency degraded.```

[the remark this answers is not in the document]
Comment (dianpopa) [reply tests/integration_tests/performance/test_snapshot_perf.py:62]: I will try to re run 1000 iterations and see where the threshold lies.

[the remark this answers is not in the document]
Comment (sandreim) [reply tests/integration_tests/performance/test_snapshot_perf.py:62]: We can assume a higher number for now and lower it once we get some good test isolation in CI to remove the variance that is causing intermittent failures now.

**Review by sandreim — APPROVED** [review]

**Review by luminitavoicu — APPROVED** [review]
LGTM

**Pull request merged**